test/integration/recruiterController.test.js

```diff
@@ -0,0 +1,53 @@
+const chai = require('chai')
+const { expect } = chai
+const chaiHttp = require('chai-http')
+
+const app = require('../../server')
+const users = require('../../models/users')
+const addUser = require('../utils/addUser')
+const cleanDb = require('../utils/cleanDb')
+
+chai.use(chaiHttp)
+
+describe('Recruiters', function () {
+  let username
+  beforeEach(async function () {
+    const userId = await addUser()
+    const { user } = await users.fetchUser({ userId })
+    username = user.username
```

Comment:
We can destructure the username here itself
`const { user: { username }} = await users.fetchUser({ userId })`

Reply:
![code1](https://user-images.githubusercontent.com/75256312/111424479-d60b6280-8717-11eb-9790-1b387d46f90d.PNG)


If I destructure in beforeEach then I am not able to access the username outside this block.
If I declare the username globally then it is  redeclaring in beforeEach and  getting  error.

Reply:
I see. Becoming JS level issue now.
What you could do is declare it as `let username` just before the `beforeEach` inside the `Recruiters` describe block. Don't assign it value just yet. This will allow the whole `Recruiters` describe block, to access the variable `username`.

Then at the time of destructuring you could assign value to it. Remove `const` while destructuring.

Hopefully this should work